libs/langchain/tests/unit_tests/chains/test_qa_with_sources.py

```diff
@@ -27,6 +27,12 @@
             "This Agreement is governed by English law.\n",
             "28-pl",
         ),
+        (
+            "According to the sources, the agreement is governed by English law.\n"
```

Comment:
with a comma after sources this would've worked before right? should we drop the comma

Reply:
Thought I had updated it, my bad! Thanks for pointing it, I've updated it now.